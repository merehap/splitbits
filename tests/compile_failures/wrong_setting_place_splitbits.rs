use splitbits::*;

fn main() {
    splitbits!(0b11011101, min=u8, "aaabbccc");
    splitbits!(0b11011101, "aaabbccc", min=u8);
}
