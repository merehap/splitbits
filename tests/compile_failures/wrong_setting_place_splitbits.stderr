error: proc macro panicked
 --> tests/compile_failures/wrong_setting_place_splitbits.rs:4:5
  |
4 |     splitbits!(0b11011101, min=u8, "aaabbccc");
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: the first argument to be a 'min' setting since three arguments were supplied

error: proc macro panicked
 --> tests/compile_failures/wrong_setting_place_splitbits.rs:5:5
  |
5 |     splitbits!(0b11011101, "aaabbccc", min=u8);
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: the first argument to be a 'min' setting since three arguments were supplied
